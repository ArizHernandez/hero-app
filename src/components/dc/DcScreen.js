import React from 'react'
import { HerosList } from '../heroes/HerosList'

export const DcScreen = () => {
  return (
    <div className="container mt-3">
      <h1 className="text-center">DC Screen</h1>
      <hr className="mt-0 mb-4"/>
      <HerosList publisher="DC Comics"/>
    </div>
  )
}
