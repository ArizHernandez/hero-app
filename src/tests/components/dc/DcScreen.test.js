import { shallow } from "enzyme";
import { DcScreen } from "../../../components/dc/DcScreen"

describe('Pruebas en <DcScreen />', () => {
  
  const wrapper = shallow(<DcScreen />)

  test('Debe cargar correctamente', () => {
    expect(wrapper).toMatchSnapshot();
  });
  
  test('Debe de cargar HerosList', () => {
    expect(wrapper.find('HerosList').exists()).toBe(true);
  });
  
})
