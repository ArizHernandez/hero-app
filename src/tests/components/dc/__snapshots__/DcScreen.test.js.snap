// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <DcScreen /> Debe cargar correctamente 1`] = `
<div
  className="container mt-3"
>
  <h1
    className="text-center"
  >
    DC Screen
  </h1>
  <hr
    className="mt-0 mb-4"
  />
  <HerosList
    publisher="DC Comics"
  />
</div>
`;
